cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -561,6 +561,10 @@ func (mh *MessageHandle) saveSuccessPoint(msg *beehiveModel.Message, info *model
 					ObjectName:       resourceName,
 				},
 			}
+			if objectSync.Spec.ObjectKind == "" {
+				klog.Errorf("Failed to init objectSync: %s, ObjectKind is empty, msg content: %v", objectSyncName, msg.GetContent())
```

Comment:
I'm wondering which message's kind/type will be empty?

Reply:
i made a mistake, configmap object metatype is set is edgecontroller downstream: https://github.com/kubeedge/kubeedge/blob/master/cloud/pkg/edgecontroller/manager/common.go#L29

Reply:
I'm wondering what kind of message's ObjectKind will be empty?